subprojects/packagefiles/libgpiod/bindings/cxx/examples/meson.build

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+executable('gpiodetectcxx', 'gpiodetectcxx.cpp',
+           include_directories: '..',
+           link_with: gpiodcxx)
```

Comment:
'.' -> gpiodcxx_includes

... or just dependencies: gpiodcxx_dep which is supposed to expose everything all together for you.